bin/migrate.sh

```diff
@@ -22,3 +22,5 @@ uploader-api-php
 for app in ${APPS}; do
     exec_container $app "bin/migrate.sh"
 done
+
+docker compose run --rm configurator bin/console configure -v
```

Comment:
As spoken, it should be a Doctrine Migration, so we can add "configurator" in `APPS` (L.16)